src/main/java/no/ccat/CcatApiApplication.java

```diff
@@ -29,6 +30,7 @@
     @Value("${springfox.documentation.swagger.v2.path}")
     private String swagger2Endpoint;
 
+
```

Comment:
Kan være greit å kjøre lint for å ikke legge til unødvendig imports eller linjerskift